bin/tachyon-start.sh

```diff
@@ -26,6 +26,8 @@ MOPT is one of:
 
 -f  format Journal, UnderFS Data and Workers Folder on master
 
+-k  Do not try to kill prior running masters and/or workers in all or local
```

Comment:
How about 'N'?  I kept it to single characters b/c the script is currently using the bash builtin of 'getopts', which (AFAIK) doesn't support long args.